frontend/src/Routers.js

```diff
@@ -5,16 +5,17 @@ import Home from './pages/home/Home';
 import Login from './pages/login/Login';
 import Signup from './pages/signup/Signup';
 import OccupancyTable from './components/tables/OccupancyTable';
+import ProtectedRoute from "./components/tables/protectedRoute/ProtectedRoute";
 
 import React from 'react';
 
 function Routers() {
   return (
     <Routes>
-        <Route path="/home" element={ <Home/> } >
-          <Route path='user' element={<UserTable />} />
-          <Route path='resource' element={<ResourceTable />} />
-          <Route path='occupancy' element={<OccupancyTable />} />
+        <Route path="/home" element={<ProtectedRoute Component={Home}/> } >
+          <Route path='users' element={<ProtectedRoute Component={UserTable} />} />
```

Comment:
why these paths are sub-paths of `/home` ?

Reply:
Because I am rendering all the table components inside home page.

Reply:
ok